values/minio/minio.gotmpl

```diff
@@ -0,0 +1,127 @@
+{{- $v := .Values }}
+{{- $vl:= $v.apps.minio }}
+{{- $k := $v.apps.keycloak }}
+{{- $minioDomain := printf "minio.%s" $v.cluster.domainSuffix }}
+{{- $rootCASecretName := "root-ca" }}
+
+mode: standalone
+auth:
+  rootUser: otomi-admin
+  rootPassword: {{ $v.otomi.adminPassword }}
+
+## OIDC configuration
+{{/* To do: add mappers to Keycloak task
+extraEnvVars:
+  - name: MINIO_IDENTITY_OPENID_CONFIG_URL
+    value: {{ $v._derived.oidcWellKnownUrl }}
+  - name: MINIO_IDENTITY_OPENID_CLIENT_ID
+    value: {{ $k.idp.clientID  }}
+  - name: MINIO_IDENTITY_OPENID_CLIENT_SECRET
+    value: {{ $k.idp.clientSecret }}
+  - name: MINIO_IDENTITY_OPENID_CLAIM_NAME
+    value: policy
+  - name: MINIO_IDENTITY_OPENID_SCOPES
+    value: openid
+  - name: MINIO_IDENTITY_OPENID_REDIRECT_URI
+    value: "https://{{ $minioDomain }}/oauth_callback"
+*/}}
+
+{{- if $v._derived.untrustedCA }}
+extraVolumes:       
+  - name: root-ca-1
+    secret:
+      secretName: root-ca
+  - name: root-ca-2
+    secret:
+      secretName: root-ca
+extraVolumeMounts:
```

Comment:
Could you add a comment in the code explaining why the same cert needs to be mounted twice?